hip-clang.docker

```diff
@@ -70,3 +71,14 @@ RUN export HIPCC_COMPILE_FLAGS_APPEND='-O3 -Wno-format-nonliteral -parallel-jobs
 # install last released miopentensile in default, install latest commits when MIOTENSILE_VER="latest"
 RUN if [ "$MIOTENSILE_VER" = "latest" ] ; then cget -p $PREFIX install ROCmSoftwarePlatform/MIOpenTensile@7568654c938d42e9a91c6b18fb382f5b978d12fd; else cget -p $PREFIX install ROCmSoftwarePlatform/MIOpenTensile@5fe0bf4a8dc59f3ab62df929297280915372ce16; fi
 
+RUN cd ~ && \
```

Comment:
@pfultz2 FYI: The library build/installation logic moved to dockerfile. When the build is finished, cmake post build stage will install the /opt/rocm/lib/libMLIRMIOpen.a

Reply:
Shouldn't it be installed into /opt/rocm/llvm?

Reply:
A few reasons make me not want to install it to /opt/rocm/llvm:
- This library comes with a different version of llvm, it does not depend on the installed ROCm version of llvm therefore it is confusing to put it to the same place
- This library exposed interface that is unrelated with any llvm library, MIOpen is its only client